addons/account/models/account_move.py

```diff
@@ -2539,7 +2540,7 @@ def _get_quick_edit_suggestions(self):
         for that partner as the default one, otherwise the default of the journal.
         """
         self.ensure_one()
-        if not self.quick_edit_total_amount:
+        if not self.quick_edit_mode or not self.quick_edit_total_amount:
```

Comment:
I don't see how this is protecting against the traceback?
If I had to guess why there is a traceback, I would assume that it is because of an invalidation or a recomputation protection because of the graph of dependencies.